### ISSUE TITLE
Silent/Autonomous Open-interpreter

### ISSUE THREAD

**Issue by vasnt** [KillianLucas/open-interpreter#984]
### Is your feature request related to a problem? Please describe.

If i understand correctly ( by watching demo from https://github.com/KillianLucas/open-interpreter/#demo), this will work ONLY after prompting. 
instead, If it can remain turned on in back-end, and observes my day to day routine tasks and suggest shortcuts for doing tasks.
Manytimes, Computer users are executing same(similar) kind of sequence of actions in routine (unconsciously)

### Describe the solution you'd like

e.g. If i am always doing tasks by clicking/action D after C & B. This new feature will suggest & create shortcut to do B--C--D altogether with a single click/action. 

There should be also dashboard/log how many (with some numbers) repetitive/steps things i am doing  & in which sequence, so that user can review it and start using newly generated shortcuts. 

### Additional context

I can imagine this feature as/like Kind of (Automated) Autohotkey/PowerAutomate (of windows or equivalent tools on MAC) on steroids.

**Comment by MikeBirdTech** [comment]
Hi @vasnt 
Your idea sounds like a cool tool but it is out of scope for OI. Here's the project's scope: https://github.com/KillianLucas/open-interpreter/blob/main/docs/ROADMAP.md#whats-not-in-our-scope

You might be able to build a project with OI playing a role in the system though. Thanks!
